### DISCUSSION TITLE
Unable to verify my website

### DISCUSSION THREAD

**Discussion by ChrisMBarr** [mastodon/mastodon#25645] [Q&A] [answered]
My website (http://chrisbarr.me) has the following line I added
```html
<a rel="me" href="https://hachyderm.io/@FiniteLooper" style="display:none;">Mastodon</a>
```
at the end of my HTML `<body>` tag.

Yet it will not add the verification for my website on my profile
![image](https://github.com/mastodon/mastodon/assets/463685/4a70d003-8426-4a47-a8f3-8632f71aec3f)

Am I missing something here?

**Comment by ianh** [comment, marked as answer]
See https://github.com/mastodon/mastodon/issues/20242.  My personal website is also HTTP-only, so this frustrates me too.

**Reply by ChrisMBarr** [reply]
Well that's what I had suspected.  It's too bad since I'm with GoDaddy which can't use LetsEnctrypt free SSL certs. Well... it can but it won't allow for auto-renewing easily so hey don't recommend using their service on GoDaddy.

I guess a work around would be to set one up temporarily until I'm verified and then remove it... but I'm not sure it's worth the trouble just to get a checkmark next to my URL.
